.github/workflows/linux_sanitizers.yml

```diff
@@ -468,3 +469,86 @@ jobs:
           name: test-results-cpp
           path: ${{ env.INSTALL_TEST_DIR }}/TEST*.xml
           if-no-files-found: 'error'
+  CPU_Functional_Tests:
+    name: CPU functional tests
```

Comment:
I suggest copying almost all configuration from the `C++ Unittests` job - like runner, `needs`, container, strategy, etc.